src/main/native/windows/process.cc

```diff
@@ -289,12 +289,12 @@ int WaitableProcess::WaitFor(int64_t timeout_msec,
     DWORD CompletionCode;
     ULONG_PTR CompletionKey;
     LPOVERLAPPED Overlapped;
-    while (GetQueuedCompletionStatus(ioport_, &CompletionCode,
-                                     &CompletionKey, &Overlapped, INFINITE) &&
-           !((HANDLE)CompletionKey == (HANDLE)job_ &&
-             CompletionCode == JOB_OBJECT_MSG_ACTIVE_PROCESS_ZERO)) {
-      // Still waiting...
-    }
+    // while (GetQueuedCompletionStatus(ioport_, &CompletionCode,
```

Comment:
I guess you need to report this as a bug to incredibuild?

Reply:
You're right. I have already contacted IncrediBuild support staff and left the address of this PR.